arrow-fx/src/main/kotlin/arrow/fx/Queue.kt

```diff
@@ -20,95 +19,232 @@ typealias QueuePartialOf<F> = Kind<ForQueue, F>
 inline fun <F, A> QueueOf<F, A>.fix(): Queue<F, A> =
   this as Queue<F, A>
 
+/** A polymoprhic effect typeclass that allows [Dequeue]'ing values from a [Queue]. */
 interface Dequeue<F, A> {
+
+  /**
+   * Take a value from the [Queue], or sementically blocks until a value becomes available.
```

Comment:
```suggestion
   * Take a value from the [Queue], or semantically blocks until a value becomes available.
```

This occurs in quite a few places.